docs/configuration/configuring-profiles.md

```diff
@@ -0,0 +1,226 @@
+# Configuring profiles
+
+Profiles are the units, where you can configure what you want to poll and then assign them to the device. The definition of profile lays in `values.yaml` file
+under the `scheduler` section.
+
+Here is the instruction of how to use profiles: [Update Inventory and Profile](../deployment-configuration/#update-inventory-and-profile). 
+
+There are two types of profiles in general:
+
+1. Static profile - polling starts when profile is added to `profiles` field in `inventory` of the device
+2. Smart profile - polling starts when configured conditions are fulfilled, and the device to poll from has `smart_profiles` enabled in inventory.
+Smart profiles are useful when we have many devices of certain kind, and we don't want to configure all of them "one by one" with static profiles.
+   In order to do so, we elect one of the fields (most commonly `sysDescr`), set the filter to match all the devices of this kind and setup polling of the profile.
+
+The template of the profile looks like following:
+
+```yaml
+scheduler:
+    profiles: |
+      #Name of profile
+      basev1:
+        # Define frequency for profile
+        frequency: 10
+        #Define condition
+        condition:
+          # Define type of condition. Allowed value field and base 
+          type: field
+          field: "SNMPv2-MIB.sysDescr"
+          # Define paterns
+          patterns:
+            - '.*STRING_TO_BE_MATCHED.*'
+        #Define varbinds to query
+        varBinds:
+          # Syntax: [ "MIB-Component", "MIB object name"[Optional], "MIB index number"[Optional]]
+          - ['SNMPv2-MIB']
+          - ['SNMPv2-MIB', 'sysName']
+          - ['SNMPv2-MIB', 'sysUpTime',0]
+```
+
+For example, we have configured two profiles. One is smart, the other one is static:
+
+```yaml
+scheduler:
+    profiles: |
+      smart_profile:
+        frequency: 10
+        condition:
+          type: field
+          field: "SNMPv2-MIB.sysDescr"
+          patterns:
+            - '.*linux.*'
+        varBinds:
+          - ['SNMPv2-MIB']
+          - ['SNMPv2-MIB', 'sysName']
+          - ['SNMPv2-MIB', 'sysUpTime',0]
+      static_profile:
+        frequency: 300
+        varBinds:
+          - ['IP-MIB']
+```
+
+If we want to enable only `static_profile` polling for host `10.202.4.202`, we will configure inventory like that:
+
+```yaml
+poller:
+    inventory: |
+      address,port,version,community,secret,security_engine,walk_interval,profiles,smart_profiles,delete
+      10.202.4.202,,2c,public,,,2000,static_profile,f,
+```
+
+If we want to enable checking `10.202.4.202` device against smart profiles, we need to set `smart_profiles` to `t`:
+
+```yaml
+poller:
+    inventory: |
+      address,port,version,community,secret,security_engine,walk_interval,profiles,smart_profiles,delete
+      10.202.4.202,,2c,public,,,2000,,t,
+```
+
+Then if the device `sysDescr` match `'.*linux.*'` filter, `smart_profile` profile is enabled.
```

Comment:
Then if the device `sysDescr` matches `'.*linux.*'` filter, then `smart_profile` profile will be enabled/active/polled (?).
here enabled - smart profiles will be enabled and if profile is matched then it will be polled